.eslintrc.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
   ],
   extends: [
     "eslint:recommended",
+    "next",
```

Comment:
nice! would probably be good to make a separate ticket to clean up which of these eslint plugins and extensions are actually necessary.